### ISSUE TITLE
Support linking to repost that can't be bridged

### ISSUE THREAD

**Issue by luxaritas** [snarfed/bridgy-fed#1543]
Personally, I would like a complete-as-possible view of my feed to be bridged. Even if a repost (or quote) can't be bridged, I would love to see Bridgy instead create a post that includes a link to the original post, if possible.

**Comment by panosda** [comment]
Yes! It's really frustrating seeing posts from bs in fedi saying stuff like "I agree with this" without any context, took me some seconds to understand what's happening. Actually, this is how Misskey and forks handle quotes in fedi itself: Since Mastodon doesn't support quotes, quotes from Misskey etc just show the url when federated to fedi platforms that don't support them.
